dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2

```diff
@@ -35,13 +37,44 @@ route-map FROM_BGP_PEER_V4 permit 10
 !
 route-map FROM_BGP_PEER_V4 permit 11
   match community allow_list_default_community
+{% if CONFIG_DB__DEVICE_METADATA and 'localhost' in CONFIG_DB__DEVICE_METADATA and 'type' in CONFIG_DB__DEVICE_METADATA['localhost'] and 'subtype' in CONFIG_DB__DEVICE_METADATA['localhost'] %}
+{% if CONFIG_DB__DEVICE_METADATA['localhost']['type'] == 'SpineRouter' and CONFIG_DB__DEVICE_METADATA['localhost']['subtype'] == 'UpstreamLC' %}
```

Comment:
Do we need to add dependencies for these in bgpcfgd?

Reply:
we already have dependency for `type` which i think should be good for `subtype` also. As this data is written at same time to config db.